docs/knowledge_base/tips/long_callbacks/index.md

```diff
@@ -0,0 +1,73 @@
+# Long Running Callbacks
```

Comment:
No need for a H1 title here. The nav bar title will be added automatically, so you will have 2 titles.
```suggestion
```